packages/0x.js/test/artifacts/AccountLevels.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Why do we commit those?

Reply:
Because they are needed to deploy the contracts before running the 0x.js tests.